src/astro/sql/operators/save_file.py

```diff
@@ -0,0 +1,86 @@
+from typing import Optional, Union
```

Comment:
Do we plan to rename `save_file` to  `export_file` (https://docs.google.com/document/d/1coJwrqm3wbCcgop_9fQSiQDdZOMLZ9r7_ENDOyO8JrQ/edit?usp=sharing) in a separate PR or should we keep this as `save_file`?

Reply:
Let's do it in a separate PR.